summerofcode/2023.md

```diff
@@ -462,6 +465,28 @@ OCI Artifacts are a set of conventions that allows us to store assets other than
 - Difficulty: Medium
 - Upstream Issue (URL): <https://github.com/kubernetes-sigs/kubebuilder/issues/3244>
 
+### Kubescape
+
+#### Design, build and document a generic security API
+
+- Description: Kubescape has the ability to submit scan results to a backend service, which can then store them and provide UIs to give insight over time as to a users' security posture. There is an implementation of this API by the commercial service that Kubescape's original creators, ARMO, runs. We would like to fully document this API, and improve it for future use cases, which may eventually have a wider scope than just Kubescape itself. If the student wants to extend the project, you could also build a sample backend to store data in-cluster using this API.
+- Expected outcome: An improved and fully-documented API for how to send scan results from a Kubescape client to a security backend service. 
+- Recommended Skills: Go, OpenAPI
+- Mentor(s): [Craig Box](https://github.com/craigbox), [Ben Hirschberg](https://github.com/slashben81)
+- Expected project size: 175h (with extension to 350h possible by building a sample backend)
```

Comment:
I am not 100% sure if the project size can be increased later when the coding starts. (deadline extension is possible, but it is a different concept; keeping the total hours the same but working less per week).

So, @craigbox @slashben81, please ensure the proposals you receive actually are matching the project size selected by applicants. (like, in this case, the applicant must select 350h if they include a sample backend in their proposal document, or vice versa).

Reply:
To clarify, this is all fine. Just a heads up for proposal reviews

Reply:
I liked that you left choosing the project size to the applicant 👍